t/basic.t

```diff
@@ -227,7 +227,7 @@ subtest_streamed(
                             call subevents => array {
                                 event Exception => sub {
                                     call error => match
-                                      qr{\Qforced die at t\E.\Qbasic.t\E.+}s;
+                                      qr{\Qforced die at \E(\./)?\Qt\E.\Qbasic.t\E.+}s;
```

Comment:
The parens are unnecessary and I'm not sure this is the right fix anyway. I think it'd be better to rewrite the regex as something like this `qr{\Qforced die at \E.*\Qbasic.t\E.+}s`

We don't really care much about the path leading up to the filename, we just want to make sure it's reporting the right exception.